src/Microsoft.Experimental.Collections/Microsoft/Collections/Extensions/HashHelpers.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
```

Comment:
nit: remove extra new line

Reply:
That's a direct copy/paste from coreclr.

Reply:
Should these remain in sync with https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/Collections/HashHelpers.cs?

If so, can you please add a comment stating where you got this from.

Reply:
Also, regarding the nit, we should remove the newline from both places :)

Other files don't have the extra new lines, for example see: https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/src/System/Numerics/Hashing/HashHelpers.cs